packages/ui/react-ui-theme/src/styles/theme.ts

```diff
@@ -54,7 +53,6 @@ export const defaultTheme: Theme<Record<string, any>> = {
   scrollArea: scrollAreaTheme,
   separator: separatorTheme,
   status: statusTheme,
-  tag: tagTheme,
```

Comment:
If you remove this here and tagTheme is no longer called by `tx`, we should probably remove `tag.ts` entirely.

Reply:
Will still use it in `Tag.tsx`

Reply:
Do we need to? It only applies a classname. I think it can be removed.